leader-election-impl/src/main/java/com/palantir/paxos/PaxosQuorumChecker.java

```diff
@@ -38,6 +38,9 @@
 
     public static final int DEFAULT_REMOTE_REQUESTS_TIMEOUT_IN_SECONDS = 5;
     private static final Logger log = LoggerFactory.getLogger(PaxosQuorumChecker.class);
+    private static final String PAXOS_MESSAGE_ERROR =
+                    "We encountered an error while trying to request an acknowledgement from another paxos node." +
+                    "This could mean the node is down, or we cannot connect to it for some other reason";
```

Comment:
nit: space after period in first sentence, and period at end of second sentence.